public/styles.css

```diff
@@ -2,4 +2,10 @@
     font-family: Arial, sans-serif;
     color: #3498db;
     text-shadow: 2px 2px #2c3e50;
+  }
+
+  .hello-from-alex {
```

Comment:
Lets say you are wanting someone to pay close attention to a particular part of your pull request, you could leave a message here for them so they'll see it. i.e. "Hey man, wasn't sure if I should use arial font here, let me know if I need to use Times New Roman instead..."